vars/skipStage.groovy

```diff
@@ -501,6 +502,14 @@ boolean call(Map config = [:]) {
             return !paramsValue('CI_RPMS_leap15_TEST', true) ||
                    skip_stage_pragma('test-leap-15-rpms', 'true') ||
                    stageAlreadyPassed()
+        case 'Test RPMs on Leap 15.4':
+            return !paramsValue('CI_RPMS_leap15.4_TEST', true) ||
+                   skip_stage_pragma('test') ||
+                   skip_stage_pragma('test-rpms') ||
+                   skip_stage_pragma('test-leap-15.4-rpms', 'true') ||
+                   docOnlyChange(target_branch) ||
+                   (paramsValue('CI_RPM_TEST_VERSION', '') != '') ||
```

Comment:
```suggestion
                   (rpmTestVersion() != '') ||
```
covers use of both `params.CI_RPM_TEST_VERSION` and `RPM-test-version:` commit pragma which will both cause the problem.

Reply:
Thanks, good catch!